nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestSplitText.java

```diff
@@ -798,6 +798,7 @@ public void testWithSplitThatStartsWithNewLine() {
     }
 
     @Test
+    @Ignore // temporary, fixing it
```

Comment:
No, didn't want to hold the PR as this is kind of an edge case which I hope to address after thinking about it a bit. 
Basically the assertion fails here ```splits.get(1).assertContentEquals("");``` since it now comes it as new line. But I also wonder why is this a split to begin with with no data?
So, i wanted to look some more as well as see if it is actually a bug that needs to be fixed, but didn't want to hold the PR since regardless this is an edge case.